asyncua/client/ua_client.py

```diff
@@ -129,12 +129,13 @@ def _send_request(self, request, timeout=1, message_type=ua.MessageType.SecureMe
         self.transport.write(msg)
         return future
 
-    async def send_request(self, request, timeout=10, message_type=ua.MessageType.SecureMessage):
+    async def send_request(self, request, timeout=None, message_type=ua.MessageType.SecureMessage):
         """
         Send a request to the server.
         Timeout is the timeout written in ua header.
         Returns response object if no callback is provided.
         """
+        timeout = self.timeout if timeout is None else timeout
```

Comment:
Alternatively (and more readable): `timeout = timeout or self.timeout`

Reply:
I disagree. It will ignore `timeout=0` which is valid value meaning no timeout. Used in the `UaClient.publish()` method

I think this is better (but request has been already merged):
```suggestion
        if timeout is None:
            timeout = self.timeout 
```

Reply:
Ah true ok, didn't realize `0` was valid.

Reply:
We can change it if you have a better solution.

Reply:
The commit is fine for me. but if you think it's lack in readability, please change it to more clear.